packages/commonwealth/vite.config.ts

```diff
@@ -0,0 +1,124 @@
+import react from '@vitejs/plugin-react-swc';
+import path from 'path';
+import { Alias, defineConfig, loadEnv } from 'vite';
+import { createHtmlPlugin } from 'vite-plugin-html';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+const projectRootDir = path.resolve(__dirname);
```

Comment:
is working from the packages directory `/packages/commonwealth`, but not from project root

Reply:
Great catch! Not quite sure how to solve this, any suggestions? Basically we need it to point to the packages/commonwealth root not the project root. @Rotorsoft can you weigh in here?